content/recipes/findability/wikiDataAccess.md

```diff
@@ -0,0 +1,54 @@
+:intended_audience: bioinformatician, data_scientist, data_engineer
+:has_executable_code: nope
+:recipe_name: Gaining  
+```` 
+## Main Objectives
+
+
+The primary purpose of this recipe is:
+
+> How to gain access to Wikidata
+> What restrictions are in place for a Wikidata account
+> Wikidata and RDF
+> Wikidata and SPARQL
+
+## Requirements
+
+---
+
+## Main Content
+
+### Creating an account
+The first step to gaining access to Wikidata is to register an account at (https://www.Wikidata.org/wiki/Wikidata:Main_Page)
+
+
+### Initial Restrictions
+After creating an account, large-scale edits are restricted until an account has been deemed a trusted account. Fifty edits are required for an account to be considered to be trusted
+
+## Conclusion
+In this recipe, we have gone over:
+1) How to create a Wikidata account
+2) Restrictions of editing
+
+
+## Authors
+
+
+| Name                                                                                                                                                                                                                                       | Orcid                                                                                                                        | Affiliation                           | Type                                                                              |                                                              Elixir Node                                                              | Credit Role
+|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------|---------------------------------------|-----------------------------------------------------------------------------------|:-------------------------------------------------------------------------------------------------------------------------------------:|:----------------:|
+| <div class="firstCol"><a target="_blank" href='https://github.com/'><img class='avatar-style' src='https://avatars.githubusercontent.com/no_github'></img><div class="d-block">Zachary Warnes</div></a>  </div>         | <a target="_blank" href='https://orcid.org/0000-0000-0000-0000'><i class='fab fa-orcid fa-2x text--orange'></i></a> | Maastricht University     | <i class="fas fa-graduation-cap fa-1x text--orange" alt="Academic"></i> | <img class='elixir-style' src='/the-fair-cookbook/_static/images/logo/Elixir/ELIXIR-UK.svg' ></img> | Writing – Review & Editing, Conceptualization
+
+---
+
+## License
+
+This page is released under the Creative Commons 4.0 BY license.
+
+<a href="https://creativecommons.org/licenses/by/4.0/"><img src="https://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png" height="20"/></a>
```

Comment:
Replace this with the new ```license_fairplus``` template used in this commit: https://github.com/FAIRplus/the-fair-cookbook/commit/c7d0076524ba88681022229928a88611e9d8af31

Reply:
Use CC-BY, but you can find all values here: https://github.com/FAIRplus/the-fair-cookbook/blob/migrating/_ext/global_variables_fairplus.py#L153

Reply:
Note to self Add (Zachary as key)
````{authors_fairplus}
Zachary: Writing - Original Draft
````

Reply:
````{license_fairplus}
CC-BY-4.0
````